sites/footers/no-footer-contextual-fr.md

```diff
@@ -0,0 +1,45 @@
+---
+altLangPage: no-footer-contextual-en.html
+breadcrumbs:
+ - title: Pied de page
+   link: sites/footers/footers-fr.html
+dateModified: 2023-01-16
+includes:
+  footer: edge
+language: fr
+noFooterContextual: true
+secondlevel: false
+share: true
+title: Bande principale et la bande sous pied de page
+titleH1: Afficher la bande principale et la bande sous pied de page
+---
+<div class="wb-prettify all-pre hide"></div>
+
+{% include alert-softlaunch.html component="site footer" version="4" %}
+
+Le pied de page général (à l'échelle du site) situé au bas de chaque page Web contient jusqu'à 3 bandes :
+* bande contextuelle
+* bande principale
+* bande sous pied de page
+
+La bande contextuelle est facultative sur tous les types de page (régulière, transactionnelle, de campagne).
```

Comment:
In the next paragraph we do mention that this example doesn't have a contextual band. Also we could change the intro sentence to mention that the footer "can contain" up to 3 bands instead of just saying "contains".